### PR TITLE
Tackle -805: Git configuration for MTA 6.0

### PR DESCRIPTION
**Pull request by CarmiWisemon**
MTA 6.0

Resolves: https://issues.redhat.com/browse/TACKLE-805

Preview: https://deploy-preview-565--windup-documentation.netlify.app/docs/web-console-guide/master/index.html#configuring-git-repositories

### CODE REVIEW

File: docs/topics/mta-web-administrator-view.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * docs/getting-started-guide/master.adoc
+
+## Administrator view
+The administrator view is intended to be used by administrators to set up the Tackle instance environment.
+![](/Tackle2/Views/AdminMenu.png)
+
+### Credentials
+This management module enables administrators to create and manage credentials for access to private repositories.  It also allows for the architects to assign the credentials to applications without knowing their contents.
```
Comment (rromannissen) [inline docs/topics/mta-web-administrator-view.adoc:10]: Credentials are not assigned to applications in the Credentials view, but in the Application Inventory view.

File: docs/topics/mta-web-administrator-view.adoc [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+![](/Tackle2/Views/GitConfig.png)
+* Git
+    * Consume insecure Git repositories
+![](/Tackle2/Views/SubvConfig.png)
+* Subversion
+    * Consume insecure Subversion repositories
+![](/Tackle2/Views/MavenConfig.png)
+* Maven
+    * Manage repository size
+    * Force update of dependencies
```
Comment (rromannissen) [inline docs/topics/mta-web-administrator-view.adoc:38]: I think forcing the update of Maven deps has been removed. @jortel can you confirm?

Comment (jortel) [reply docs/topics/mta-web-administrator-view.adoc]: Yes.  Removed.

File: docs/topics/mta-web-config-git-rep.adoc [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+[id="mta-web-config-git-rep_{context}"]
+= Configuring repositories
+
+This management module configures repositories used by {ProductName} ({ProductShortName}) {WebName} with the following options:
+
+* Configuring Git repositories
+* Configuring Subversion repositories
+* Configuring Maven repositories
+
+== Configuring Git repositories
```
Comment (istein1) [inline docs/topics/mta-web-config-git-rep.adoc:15]: Only git repositories is detailed here. What about Subversion/Maven?

Comment (CarmiWisemon) [reply docs/topics/mta-web-config-git-rep.adoc:15]: Subversion/Maven were assigned different Jiras and they are in different modules.

File: docs/topics/mta-web-config-git-rep.adoc [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+.Procedure
+
+. In the *Administrator* view, click *Repositories* and then *Git*.
```
Comment (RichardHoch) [inline docs/topics/mta-web-config-git-rep.adoc:19]: then click

File: docs/topics/mta-web-config-git-rep.adoc [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+. Toggle *Consume insecure Git repositories* to enable.
```
Comment (RichardHoch) [inline docs/topics/mta-web-config-git-rep.adoc:20]: add object following "enable."

**Comment by RichardHoch** [comment]
@CarmiWisemon: See notes, but otherwise LGTM.

**Pull request merged**